neural_compressor/strategy/utils/utility.py

```diff
@@ -126,6 +126,37 @@ def get_adaptor_name(adaptor):
     return ""
 
 
+def check_key_exist(data, key):
```

Comment:
Suggest moving the function into `neural_compressor/utils/utility.py`, since it is a common function that is not specific to `strategy`.